src/components/vm/consoles/VncConsole.jsx

```diff
@@ -51,14 +51,14 @@ export const VncConsole = ({
     scaleViewport = false,
     viewOnly = false,
     shared = false,
-    credentials,
     repeaterID = '',
     vncLogging = 'warn',
     consoleContainerId,
     onConnected = (element) => {},
-    onDisconnected = () => {},
+    onDisconnected = (event) => {},
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2038-5fe3bc4a-20250808-144213-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)